core/scripts/chaincli/command/keeper/root.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/command"
+)
+
+// rootCmd represents the root keeper sub-command to manage keepers
+var rootCmd = &cobra.Command{
+	Use:   "keeper",
+	Short: "Manage keepers",
+	Long:  `This command represents a CLI interface to manage keepers.`,
+}
+
+func init() {
+	command.RootCmd.AddCommand(rootCmd)
```

Comment:
Seems odd to do it this way, where the child command imports the parent command and calls `AddCommand`. The other way around (adding all commands from the parent) seems clearer. 

```golang
// in command/root.go
func init() {
  rootCmd.AddCommand(keeper.RootCommand())
}
```

We do something similar in Atlas, where the command is created via an exported function. Advantages of this is you see all the subcommands being added in one place, the parent, whereas this way you'd have to go to all the subpackages to see the subcommands being added. I'm good either way though, just a thought.

Reply:
Agree, done